tron/core/job.py

```diff
@@ -446,6 +449,7 @@ def build(self, job_config):
             self.context,
             output_path,
             self.action_runner,
+            eventbus_publish=self.eventbus_publish,
```

Comment:
Since you've used kargs for eventbus_publish, why not using kargs for all of them?

Reply:
mostly laziness, i'll fix :)